packages/@lwc/engine-core/src/framework/vm.ts

```diff
@@ -243,12 +243,6 @@ export function createVM<HostNode, HostElement>(
         renderer: Renderer;
     }
 ): VM {
-    if (process.env.NODE_ENV !== 'production') {
-        assert.invariant(
-            elm instanceof HTMLElement,
```

Comment:
i wonder why we had this assert?!
looking at [the spring20 branch](https://github.com/salesforce/lwc/blob/spring20/packages/%40lwc/engine/src/framework/vm.ts#L213), the type is the same for `elm`, so i would assume ts will catch these cases.

Reply:
Good question, I don't know either.

Reply:
Verified that this code path is only ever hit internally so we should be able to remove it.